go/felix/intdataplane/endpoint_mgr.go

```diff
@@ -207,8 +207,24 @@ func (m *endpointManager) resolveWorkloadEndpoints() error {
 			var ipStrings []string
```

Comment:
How about defining an `extraIPs` slice here, then adding the NAT IPs to it below, then the copy logic can be shared?